bin/newman.js

```diff
@@ -50,6 +50,7 @@ program
     .option('--timeout [n]', 'Specify a timeout for collection run (milliseconds)', util.cast.integer, 0)
     .option('--timeout-request [n]', 'Specify a timeout for requests (milliseconds)', util.cast.integer, 0)
     .option('--timeout-script [n]', 'Specify a timeout for scripts (milliseconds)', util.cast.integer, 0)
+    .option('--output-size [n]', 'Specify the output size limit for CLI ( in  KB )', util.cast.integerOrInfinity, 2)
```

Comment:
English here is ambiguous "output size of what? Entire CLI, body always, body in verbose"?!

Reply:
Technically, this PR limits the verbose body output only. I'll make the necessary changes to remove this ambiguity.